### ISSUE TITLE
Use Resnet18 to replace Resnet50

### ISSUE THREAD

**Issue by NeoYY** [layumi/Person_reID_baseline_pytorch#267]
Is it possible to use resnet18 instead of resnet50 as the baseline? Will there be any problems?

**Comment by layumi** [comment]
Yes. @NeoYY It is possible. You may check the pretrained model from Pytorch.
